Automattic/wp-calypso
# Reader: remove/reader/daily-post component

The Daily Post button was previously moved to `client/blocks` and the old component is no longer in use anywhere.

I've updated the location of the block in `reader/stream/post.jsx` (no longer used after Reader Refresh) just so that the build succeeds.

### To test

For good measure, head to 

http://calypso.localhost:3000/read/feeds/27030

and ensure that the Daily Post button appears as expected.

<img width="388" alt="screen shot 2016-12-16 at 11 14 33" src="https://cloud.githubusercontent.com/assets/17325/21244099/da79f9d0-c380-11e6-96bc-e53f46d1fd65.png">
